i18n/en/pop_desktop_widget.ftl

```diff
@@ -49,10 +49,10 @@ gis-extensions-label1 = Manually installed GNOME Shell extensions are disabled t
 gis-extensions-label2 = To this backup folder:
 gis-extensions-title = GNOME Shell Extensions Update
 
-gis-gestures-description = Use four finger swipe left to open Workspaces and windows overview, four fingers swipe right to open Applications, and four fingers swipe up or down to switch between workspaces. Swipe with three fingers to switch between windows.
+gis-gestures-description = Use four finger swipe left to open the Windows and Workspaces Overview, four fingers swipe right to open Applications, and four fingers swipe up or down to switch between workspaces. Swipe with three fingers to switch between windows.
```

Comment:
This is to make the name consistent with the same name on line 6.

By adding "the", it makes it more clear that we are talking about a single named feature. Before, it could easily be misread as "open all the workspaces, then open the windows overview".